pages/common/routers/related-tasks.js

```diff
@@ -49,7 +49,7 @@ module.exports = getQuery => {
       next();
     },
     locals: (req, res, next) => {
-      res.locals.static.content = merge({}, res.locals.static.content, content);
+      res.locals.static.content = merge({}, content, res.locals.static.content);
```

Comment:
Need to change the merge order here, otherwise the content from relatedTasks overrides `fields.establishment.label` and we can't change it in `project/read/content/index.js`

Reply:
Makes sense. This way round sets defaults for the related tasks, but prefers page-local content definitions.